docs/sections/cli_features.md

```diff
@@ -98,6 +98,46 @@ Workflow docker-data-science has been added successfully.
 This will download the contents of the workflow and all its 
 dependencies to your project tree.
 
+
+## Searching for actions
+
+The popper CLI is capable of searching for premade actions that
+you can use in your workflows.
+
+You can use the `popper search` command to search for actions
+based on a search keyword. For example, to search for npm based actions,
+you can simply run:
+
+```bash
+$ popper search npm
+Matched actions :
+
+> actions/npm
+```
+
+Additionally, when searching for an action, you may choose to include
+the contents of the readme in your search by using the `--include-readme`
+flag.
+
+Once `popper search` runs, it caches all the metadata related to the search.
+So, to get the latest releases of the actions, you might want to update the
+cache using the `--update-cache` flag.
+
+By default, popper searches for actions from [popperized](https://github.com/popperized)
```

Comment:
we can link to the file here, as the list of default orgs/repos might change over time